### PR TITLE
feat: 🎸 Allow creation of nft collection with custom nft type

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Allows creation of nft collection with custom nft type string. Earlier error was thrown here

### Breaking Changes

NA

### JIRA Link

DA-1335

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/procedures/createNftCollection.ts [hunk 1]
```diff
@@ -396,23 +409,26 @@ async function prepareStorageForCustomType(
...
-          'createNftCollection procedure was given a custom type string that does not have a corresponding ID. Register the type and try again',
-        data: { nftType },
-      });
+      const rawCustomAssetTypeId = await context.polymeshApi.query.asset.customTypeIdSequence();
+      const nextCustomAssetTypeId = u32ToBigNumber(rawCustomAssetTypeId).plus(1);
+      customTypeData = {
+        rawId: bigNumberToU32(nextCustomAssetTypeId, context),
+        rawValue,
+        isAlreadyCreated: false,
+      };
```
Comment (prashantasdeveloper) [inline src/api/procedures/createNftCollection.ts:424]: Like we do for Asset ID (where we calculate the next Asset ID to be generated and then batch the other txs with this ID), I have calculated the next custom asset type ID to be generated and later used the same while creating Asset (After registering the custom asset ID first)

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**